recon/src/libp2p.rs

```diff
@@ -387,10 +404,11 @@ where
         Ok(Handler::new(
             peer,
             connection_id,
-            // Start synchronizing interests
+            // Start synchronizing peers
             handler::State::RequestOutbound {
-                stream_set: StreamSet::Interest,
+                stream_set: StreamSet::Peer,
```

Comment:
If the peer doesn't yet support the peers protocol, this won't fail/close right? it will move on to interests and be aok?

Reply:
Yes I have tested this against live nodes as well as in unit tests